manifests/t/trinsic/okapi/1.2.1/trinsic.okapi.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS1.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: trinsic.okapi
+PackageVersion: 1.2.1
+PackageLocale: en-US
+Publisher: Trinsic (Streetcred ID)
+PublisherUrl: https://github.com/trinsic-id/okapi/
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Trinsic Okapi
```

Comment:
```suggestion
PackageName: okapi
```

Reply:
This must match Control Panel.

Reply:
Will change in control panel to `Trinsic Okapi`, in our experience, `okapi` isn't unique.